packages/module/build.js

```diff
@@ -12,6 +12,8 @@ const build = (selector) => {
     name: 'customFormat',
     formatter: function ({ dictionary, file, options }) {
       const { outputReferences } = options;
+      const alphaSort = (a, b) => sortByName(a, b) * -1;
+      dictionary.allTokens = dictionary.allTokens.sort(alphaSort).sort((tokens) => sortByReference(tokens));
```

Comment:
IIRC, we decided we don't have to sort by reference since these are CSS vars and a CSS var can be referenced before it's defined as long as it's defined somewhere in the same scope.

```suggestion
      dictionary.allTokens = dictionary.allTokens.sort(alphaSort).sort(tokens);
```